docs/guides/deployment/overriding-job-variables.md

```diff
@@ -0,0 +1,177 @@
+# Deeper Dive: Overriding Work Pool Job Variables
+
+As described in the [Deploying Flows to Work Pools and Workers](/guides/prefect-deploy/) guide, there are two ways to deploy flows to work pools: using a `prefect.yaml` file or using the `.deploy()` method.
+
+**In both cases, you can override job variables on a work pool for a given deployment.**
+
+While exactly _which_ job variables are available to be overridden depend on the type of work pool you're using at a given time, this guide will explore some common patterns for overriding job variables in both deployment methods.
+
+## Background
+First of all, what are _"job variables"_?
+
+Job variables are infrastructure related values that are configurable on a work pool, which may be relevant to how your flow run executes on your infrastructure.
+
+<hr>
+
+Let's use `env` - the only job variable that is configurable for all work pool types - as an example.
+
+When you create or edit a work pool, you can specify a set of environment variables that will be set in the runtime environment of the flow run.
+
+For example, you might want a certain deployment to have the following environment variables available:
+```json
+{
+  "EXECUTION_ENV": "staging",
+  "MY_NOT_SO_SECRET_CONFIG": "plumbus",
+}
+```
+
+So, as opposed to hardcoding these values into your work pool in the UI and making them available to all deployments associated with that work pool, you can override these values on a _per-deployment basis_.
+
+Let's look at how to do that.
+
+## How to override job variables
+Say we have the following repo structure:
+```
+» tree
```

Comment:
Not everyone will know `tree`, so maybe leave that off but keep the tree viz

Reply:
imo this is fine, its obvious what its doing and if we educate a person about a useful command then that doesn't hurt

Reply:
you win